Modules/NetworkSecurityGroups/2.0/Scripts/enable.flow.logs.ps1

```diff
@@ -0,0 +1,62 @@
+[CmdletBinding()]
+    param (
+        [Parameter(Mandatory=$true)]
+        [string]
+        $SubscriptionId,
+        [Parameter(Mandatory=$true)]
+        [string]
+        $NetworkWatcherRegion,
+        [Parameter(Mandatory=$true)]
+        [string]
+        $NetworkSecurityGroupId,
+        [Parameter(Mandatory=$true)]
+        [string]
+        $DiagnosticStorageAccountId,
+        [Parameter(Mandatory=$true)]
+        [string]
+        $WorkspaceId,
+        [Parameter(Mandatory=$true)]
+        [string]
+        $LogAnalyticsWorkspaceId,
+        [Parameter(Mandatory=$true)]
+        [string]
+        $WorkspaceRegion
+    )
+
+try {
+    $currentSubscriptionId = Get-AzContext | Select-Object "Subscription"
+
+    if ($null -ne $currentSubscriptionId -and `
+        $currentSubscriptionId.Subscription.Id -eq $SubscriptionId) {
+        Set-AzContext -Subscription $SubscriptionId
```

Comment:
Add "| Out-Null" to Set-AzContext. Otherwise, the resultant of Set-AzContext becomes part of the output. This was based on testing using this script.

Reply:
Done